index.html

```diff
@@ -8,17 +8,52 @@
     <title>Book Tracker</title>
     <link rel="stylesheet" href="./css/style.css">
 </head>
+
 <body>
-    <div class="heading-container">
-        <h2>Your Reading List:</h2>
-    </div>
-    <div id="collection" class="reading-collection"></div>
-    <div class="input-section"> 
-        <input id="title" class="input-title" placeholder="Book Title">
-        <input id="author" class="input-author" placeholder="Book Author">
-        <input type="submit" id="btnAdd" class="button">
-        <p class="error-message" id="error"></p>
-    </div>
+    <header class="nav-section">
+        <!-- <div></div> -->
+        <div class="nav-logo" id="logo">
+            <h1>Awesome Books</h1>
+        </div>
+        <nav class="nav-bar" id="navigation">
+            <h3><a href="#" class="nav-list active-tab" data-tab="index">List</a></h3>
+            <h3><a href="#" class="nav-list" data-tab="add_book">Add New</a></h3>
+            <h3><a href="#" class="nav-list" data-tab="contact">Contact</a></h3>
+        </nav>
+    </header>
+    <main>
+        <section id="index" class="section b-tab active-tab" aria-label="section">
+            <div class="heading-container">
+                <h2>Your Reading List:</h2>
+                <div class="date" id="date"></div>
```

Comment:
- [ ] It's just a small thing that but it will make your UI nice. For now, you have displayed today's date below the title whereas you should display it below the navigation bar so that it looks good and also eye-catching.

### Your design

![Screenshot from 2021-08-13 17-01-09](https://user-images.githubusercontent.com/22434660/129351034-9466bc4b-87a8-42fa-b197-6d6e9c7e0ed6.png)


### Wireframe design

![Screenshot from 2021-08-13 17-01-21](https://user-images.githubusercontent.com/22434660/129351072-75b6100a-9787-492f-8094-b3d6cb99504f.png)